internal/datasystem/fdv1_datasystem.go

```diff
@@ -0,0 +1,146 @@
+package datasystem
```

Comment:
This whole file implements the status quo, pre-FDv2. 

It refactors out a bunch of config logic that used to live in the SDK client, stuffing it behind the new `DataSystem` interface. 

Since the `DataSystem` interface offers some new capabilities, it also implements those capabilities in terms of the old system. I'll call that out in the comments.